crates/rome_js_formatter/src/js/expressions/call_arguments.rs

```diff
@@ -14,24 +21,239 @@ impl FormatNodeFields<JsCallArguments> for FormatNodeRule<JsCallArguments> {
+                                    .entry(&format_with(|f| {
+                                        write!(
+                                            f,
+                                            [group_elements(&format_args![last, expand_parent()])]
+                                        )
+                                    }))
+                                    .finish()
+                            }),
+                            r_leading_trivia,
+                            r_paren,
+                            r_trailing_trivia
+                        ]
+                    )
+                }
+            });
+
+            // This is the version of where all the arguments are broken out
+            let all_arguments_expanded = format_with(|f| {
+                // this formatting structure replicates what we have inside the `format_delimited`
+                // function, but here we use a different way to print the trailing separator
+                write!(
+                    f,
+                    [
+                        &l_leading_trivia,
+                        &l_paren,
+                        &group_elements(&format_args![format_with(|f| {
+                            write!(
+                                f,
+                                [
+                                    &soft_block_indent(&format_args![
+                                        &l_trailing_trivia,
+                                        format_with(|f| {
+                                            fmt_arguments_multi_line(
+                                                separated.iter(),
+                                                separated.len(),
+                                                f,
+                                            )
+                                        }),
+                                        r_leading_trivia,
+                                        soft_line_break()
+                                    ]),
+                                    &r_paren
+                                ]
+                            )
+                        })]),
+                        &r_trailing_trivia
+                    ]
+                )
+            })
+            .memoized();
+
+            write!(
+                f,
+                [best_fitting![
+                    format_args![
+                        l_leading_trivia,
+                        l_paren,
+                        l_trailing_trivia,
+                        group_elements(&format_args![format_with(|f| {
+                            fmt_arguments_multi_line(separated.iter(), separated.len(), f)
+                        })]),
+                        r_leading_trivia,
+                        r_paren,
+                        r_trailing_trivia
+                    ],
+                    format_args![formatted],
+                    format_args![all_arguments_expanded]
```

Comment:
I believe this should work

```suggestion
                    formatted,
                    all_arguments_expanded
```